Gaijinizer patch/Map003.txt

```diff
@@ -1227,10 +1227,10 @@ win if you are not lucky enough.#MTLed
 \\C[2]ＳＰコストが馬鹿にならないのは、お前ももう見ただろう？
 しかも効果が強力な分、消費気力も多い。
 > CONTEXT: Map003/events/1/pages/2/275/Dialogue/クッション君
-I can remember the powerful magic of derivation
-\\C[2]Have you already seen that the SP cost is not
-stupid? Moreover, because the effect is strong,
-consumption is also high.#MTLed
+You can try learning more powerful magic but
+\\C[2]you must have seen the SP cost of those
+spells is nothing to be sneezed at. You'll do
+more damage but use way more SP.#Ron
```

Comment:
Flow:
_**You can try learning the more powerful
magic, but \\C[2]the SP cost of those spells
is nothing to sneeze at. You'll do more
damage but use way for Vitality.**_